packages/synapse-react-client/src/lib/containers/trash/TrashCanList.tsx

```diff
@@ -238,7 +238,8 @@ export function TrashCanList() {
             )}
             <div style={{ margin: 'auto' }} />
             <Button
-              variant="danger"
+              variant="contained"
+              color="error"
```

Comment:
Use Mui Buttons in TrashCanList so "Delete" button color matches the WarningDialog confirm button color